.github/workflows/builder_go_slsa3.yml

```diff
@@ -282,7 +291,7 @@ jobs:
   #           Upload binaries and provenances as assets             #
   #                                                                 #
   ###################################################################
-  upload-assets:
+  upload-tag:
```

Comment:
what if this is run with `upload-url` specified too? Should we double check that it is not?

Reply:
I've left it as-is to keep it simple.
If `upload-assets: true`, always upload for tags
if `upload-url` is also set, upload to the URL. This way is a user wants to upload to a different url from the tag, they can. And if they pass both, we will upload twice... which still works.

Wdut?